website/src/components/Schema/index.js

```diff
@@ -138,16 +142,35 @@ const Schema = ({ schema, menuRef }) => {
     },
     {
       Header: 'Notes & Scripts',
-      Cell: ({ original: { meta, restrictions } }) => {
+      Cell: ({ original: { meta, restrictions, name } }) => {
         const script = restrictions && restrictions.script;
         return (
           <div>
             {meta && meta.notes && <div>{meta.notes}</div>}
             {script && (
-              <Button variant="secondary" size="sm">
+              <Button
+                variant="secondary"
+                size="sm"
+                onClick={() => {
+                  toggleHTMLOverflow();
+                  setModalVisibility(!modalVisibility);
```

Comment:
maybe we should just have a common `useModalState` hook exported from `website/src/pages/dictionary/index.js` which wraps `useState(false)` and composes `toggleHTMLOverflow` into its setter. That way components can call `setModalVisibility` without knowing about `toggleHTMLOverflow`